src/main/java/address/sync/RemoteService.java

```diff
@@ -388,7 +388,7 @@ private long getRateResetFromHeader(HashMap<String, String> header) {
     private List<Person> convertToPersonList(List<CloudPerson> cloudPersonList) {
         List<Person> convertedList = new ArrayList<>();
         cloudPersonList.stream()
-                .forEach(CloudPerson -> convertedList.add(convertToPerson(CloudPerson)));
+                .forEach(cloudPerson -> convertedList.add(convertToPerson(cloudPerson)));
 
         return convertedList;
```

Comment:
```java
return cloudPersonList.stream()
                      .filter(this::convertToPerson)
                      .collect(Collectors.toCollection(Arraylist::new));
```
would be slightly more efficient; doing `ArrayList::add` n times means that there may be multiple resizings and recopying of the array. Also it feels nice and clearer to use more of the stream methods